packages/lib-user/src/components/UserHome/components/RecentSubjects/RecentSubjects.js

```diff
@@ -59,23 +49,23 @@ function RecentSubjects({
           </Box>
         )}
         {!isLoading && recents?.length
-          ? recents.slice(0, 10).map(classification => {
-              const subjectMedia = classification.locations.map(
+          ? recents.map(recent => {
+              const subjectMedia = recent?.locations?.map(
```

Comment:
Changing this variable name to `recent` instead of `classification` was suggested [here](https://github.com/zooniverse/front-end-monorepo/pull/6112#discussion_r1630024726), because the object is a Recent resource _from_ panoptes, not a Classification resource typically sent from lib-classifier _to_ panoptes.